packages/framework/src/Component/Domain/DomainIconProcessor.php

```diff
@@ -33,21 +29,17 @@ public function __construct(
      * @param string $filepath
      * @param string $domainImagesDirectory
      */
-    public function convertToDomainIconFormatAndSave($domainId, $filepath, $domainImagesDirectory)
-    {
-        $resizedImage = $this->imageProcessor->resize(
-            $this->imageProcessor->createInterventionImage($filepath),
-            static::DOMAIN_ICON_WIDTH,
-            static::DOMAIN_ICON_HEIGHT,
-            static::DOMAIN_ICON_CROP,
-        );
-        $resizedImage->encode(ImageProcessor::EXTENSION_PNG);
-
+    public function convertToDomainIconFormatAndSave(
+        int $domainId,
+        string $filepath,
+        string $domainImagesDirectory,
+    ): void {
         $targetFilePath = $domainImagesDirectory . '/' . $domainId . '.' . ImageProcessor::EXTENSION_PNG;
```

Comment:
Does this mean when I upload JPG file, it will only be renamed to PNG? 
Is it ok?

edit: oh, now I see only PNG format is allowed in form type...  In that case, should we assume this will always be true (only PNG is allowed)? 

Nitpick: maybe the method should be named just `saveIcon`? it's not converting anymore, or?

Reply:
good point with the naming. Also, I have added check for the mime type into the method, just to be sure it works as expected regardless the form type constraints. 86ebaab209